app/tests/spec/views/settings.js

```diff
@@ -329,40 +541,88 @@ function (chai, $, sinon, View, RouterMock, TestHelpers,
             });
         });
       });
-    });
 
-    describe('communication preferences link', function () {
-      it('is visible if enabled', function () {
-        sinon.stub(view, 'isUserAuthorized', function () {
-          return p(true);
+      describe('hide success', function () {
+        it('displaySuccessUnsafe', function (done) {
+          view.SUCCESS_MESSAGE_DELAY = 5;
+          var spy = sinon.spy(view, 'hideSuccess');
+          view.render()
+            .then(function () {
+              view.displaySuccessUnsafe('hi');
+              setTimeout(function () {
```

Comment:
If this form works (I have not tried it), and you use it, make sure to remove the `done` from the `it` function signature, and add a `return` before `view.render`.